lib/spack/spack/test/concretize.py

```diff
@@ -1354,7 +1354,8 @@ def test_concrete_specs_are_not_modified_on_reuse(
         # when reused specs are added to the mix. This prevents things
         # like additional constraints being added to concrete specs in
         # the answer set produced by clingo.
-        s = spack.spec.Spec(spec_str).concretized(reuse=True)
+        with spack.config.override("concretizer:reuse", True):
```

Comment:
This test now needs the `config` fixture

Reply:
yep thanks